manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -160,6 +160,10 @@ spec:
               value: "true"
             - name: REPAIR_SIDECAR_ANNOTATION
               value: "sidecar.istio.io/status"
+            {{- if and .Values.ambient.enabled .Values.ambient.shareHostNetworkNamespace }}
```

Comment:
This condition is inverted I think, needs a `not`.

Rather concerningly, no e2e tests fail at all from this! I guess we do not have anything testing readiness probes + STRICT mode which should be the only host network usage.

Reply:
Yes, definitely a mistake.
I have fixed it now and performed a live deployment to verify that ipsets and iptables rules don't end up in the wrong namespace.
I suspect the reason why the tests did not fail is that, while we have integration tests that deploy with PA in strict mode, the PA is deployed after the application becomes ready and is deleted before the liveness probe fails for long enough to trigger a restart.
I will open an issue to keep track of this